concrete/blocks/image/form.php

```diff
@@ -78,7 +78,36 @@
 </fieldset>
 
 <fieldset>
-    <legend><?php echo t('Resize Image'); ?></legend>
+    <legend><?php echo t('Alignment'); ?></legend>
+
+    <div class="form-group">
+        <div class="radio">
+            <label>
+                <?php
+                echo $form->radio('alignImage', 0, $alignImage);
+                echo t('Left');
+                ?>
+            </label>
+        </div>
+        <div class="radio">
+            <label>
+                <?php
+                echo $form->radio('alignImage', 1, $alignImage);
+                echo t('Center');
+                ?>
+            </label>
+        </div>
+        <div class="radio">
+            <label>
+                <?php
+                echo $form->radio('alignImage', 2, $alignImage);
+                echo t('Right');
+                ?>
+            </label>
+        </div>
+    </div>
+
```

Comment:
Conceded and fixed. This sure makes the form luxuriantly padded!

A git question, if I may: do I need to close this pull request and initiate another from scratch, to reflect these changes?  Or is there a way to get newly-committed changes to my branch to apply to the current pull request here? (Please excuse my ignorance; I'm a new git.)